backend/api/v1/v1_profile/tests/mixins.py

```diff
@@ -21,21 +22,14 @@ class ProfileTestHelperMixin:
     ROLE_SUPER_ADMIN = 0
     ROLE_ADMIN = 1
     ROLE_APPROVER = 2
```

Comment:
There's no role approver, if you meant admin that has approver access to a form for a TEST, then make it clear (e.g IS_SUPER_ADMIN, IS_EDITOR, IS_READER, IS_APPROVER